qiita_pet/test/test_admin_processing_job_handlers.py

```diff
@@ -45,5 +45,19 @@ def test_get_missing_argument(self):
                       response.body.decode('ascii'))
 
 
+class TestSampleValidation(BaseAdminTests):
+    def test_get(self):
```

Comment:
As mentioned during the meeting, a good unittest should canvas the common things that might break your method/function. For example, calling post without one or both of the post_args, or giving a value for qid or snames that is obviously bad. This confirms that error-handling exists and is working properly.

Reply:
Added a unit test to check error handling of post with invalid qid